core/server/worker/src/main/java/alluxio/worker/block/UnderFileSystemBlockReader.java

```diff
@@ -119,7 +117,7 @@ public static UnderFileSystemBlockReader create(UnderFileSystemBlockMeta blockMe
   private UnderFileSystemBlockReader(UnderFileSystemBlockMeta blockMeta, boolean positionShort,
       BlockStore localBlockStore, UfsManager ufsManager, UfsInputStreamManager ufsInstreamManager)
       throws IOException {
-    mInitialBlockSize = ServerConfiguration.getBytes(PropertyKey.WORKER_FILE_BUFFER_SIZE);
+    mInitialBlockSize = blockMeta.getBlockSize();
```

Comment:
Are there any implications to this change?

Reply:
I don't think there is any downside here. @calvinjia  ?

Reply:
Doesn't`read()` method should never call `requestSpace()` after this change here?

Otherwise, each read call would allocate more space on top of the initial full block size. However, requested spaces wouldn't leak because the block-store would be updated with the physical file size as part of the commit.

Reply:
@ggezer  you are correct. `requestspace()` should not be called any more to load this block from UFS.

Reply:
This condition in `read` looks broken, hence always calling `requestSpace`.

>>> if (mBlockWriter != null && mBlockWriter.getPosition() < mInStreamPos) {
  mLocalBlockStore.requestSpace(mBlockMeta.getSessionId(), mBlockMeta.getBlockId(),
            mInStreamPos - mBlockWriter.getPosition());